github/MainClass.py

```diff
@@ -853,14 +853,18 @@ def get_access_token(self, installation_id, user_id=None):
             )
         elif response.status_code == 403:
             raise GithubException.BadCredentialsException(
-                status=response.status_code, data=response.text
+                status=response.status_code,
+                data=response.text,
+                headers=response.headers,
             )
         elif response.status_code == 404:
             raise GithubException.UnknownObjectException(
-                status=response.status_code, data=response.text
+                status=response.status_code,
+                data=response.text,
+                headers=response.headers,
             )
         raise GithubException.GithubException(
-            status=response.status_code, data=response.text
+            status=response.status_code, data=response.text, headers=response.headers
```

Comment:
why is this exception not formatted as the above two?
I would expect to have either 3 lines in all cases or only one line.

Reply:
It is probably not long enough to _require_ newlines and the linter broke the above into multiple lines but not this one.

So you are saying this should intentionally be split into multiple lines to have the same formatting as above, though it is not required.

Reply:
sure, it is only visual and not functional!

If the formatting is done automatically, I am fine with it. If it is not I would prefer to have equal coding equally formated.

Reply:
I'm not a fan of it either, but that's how it is in this [black](https://github.com/psf/black), [black](https://github.com/PyGithub/PyGithub/blob/001970d4a828017f704f6744a5775b4207a6523c/.pre-commit-config.yaml#L2-L5) world. ;)